main.js

```diff
@@ -1090,6 +1090,22 @@ ipc.on('set-media-permissions', (event, value) => {
   }
 });
 
+// Loki - Auto updating
+ipc.on('get-auto-update-enabled', event => {
```

Comment:
can we get the `-enabled` out of there? `get-auto-update_setting` would be an improvement (then saying get-auto-update-enabled is disabled)